staging/src/k8s.io/client-go/features/envvar.go

```diff
@@ -73,6 +77,10 @@ type envVarFeatureGates struct {
 	// readEnvVarsOnce guards reading environmental variables
 	readEnvVarsOnce sync.Once
 
+	// readEnvVars holds the boolean value which
+	// indicates whether readEnvVarsOnce has been called.
+	readEnvVars *atomic.Value
```

Comment:
atomic.Bool

Reply:
a cargo cult, same applies to `atomic.Value`